integration-testing/rnode_testing/casper_propose_and_deploy.py

```diff
@@ -49,15 +52,15 @@ def run(config, network):
 
     token_size = 20
 
-    contract_name = 'contract.rho'
+    contract_path = 'test/casper_propose_and_deploy/contract.rho'
```

Comment:
Here you should use the `rnode_testing.resources.file_path` instead of hardcoding the path. This way we are sure that the resources are organized in the same structure as the test files.

I know that the `contract.rho` is no longer in the right position after the last refactorings and should be moved. This shows exactly the problem which `rnode_testing.resources` is addressing.

Reply:
```suggestion
    contract_path = resources.file_path('test/casper_propose_and_deploy/contract.rho')
```
like this?

Reply:
It should be ```contract_path = resources.file_path('contract.rho')```...which says "the resource 'contract.rho' of this test"

Then you have to move the file to the expected path which I think it is `resources/casper_propose_and_deploy/contract.rho`

Reply:
If the resources.file_path would work correctly the "contract.rho" should be in `resources/casper_propose_and_deploy/contract.rho` but since there is a bug in `resources.file_path` implementation I'm happy with the changes as it is now